tests/integration-tests/tests/common/compute_logs_common.py

```diff
@@ -17,11 +17,15 @@
 @retry(
     retry_on_exception=lambda exception: isinstance(exception, RemoteCommandExecutionError),
     wait_fixed=seconds(30),
-    stop_max_delay=minutes(10),
+    stop_max_delay=minutes(15),
 )
-def wait_compute_log(remote_command_executor):
+def wait_compute_log(remote_command_executor, expected_num_nodes=1):
```

Comment:
ok keeping this for this PR. But we should remove these logs now that we have CW, hence updating these tests.